shopyo/config.py

```diff
@@ -3,35 +3,27 @@
 base_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Maybe just some comments stating not to touch this file but add infos in instance/config.py

Reply:
I can add that. But they are allowed to change this file as long as it is not a private change (i.e. they are okay with it being visible)